3-estructura/README.md

```diff
@@ -1,3 +1,139 @@
+
+### CLASIFICAR EL CONTENIDO 
+
+Tras la realización del cardsorting y en vista de los resultados obtenidos, hemos llegado a diferentes conclusiones:
+
+Por un lado, en lo que a esquemas organizativos se refiere, tenemos claro que utilizaremos un esquema ambiguo o subjetivo. Nuestra página web va a ser de actualización continua. Es decir, los contenidos van a ir actualizándose de forma periódica, por lo que no tiene sentido implantar un esquema exacto y no dinámico. 
+
+Concretamente, será un esquema de contenido por temas, siendo estos los especificados en el apartado anterior: Noticias, Reportajes, Entrevistas...
+
+Somos conscientes de que este tipo de esquema es más complicado de administrar, teniendo en cuenta que para diferentes personas diferentes noticias pueden tener diferentes clasificaciones. 
+El ejemplo más claro es lo que ocurrió en el card sorting entre los alumnos de aquí y de fuera.
+
+En cuanto a la estructura otganizativa de nuestra página web, utilizaremos un modelo de base de datos. Es decir, tenemos el contenido antes de tener nuestra estructura hecha, por lo que la estructura será claramente de abajo arriba.
+Todo ello llega impuesto en cierta medida también por los diferentes ámbitos en los que nos movemos (radio, audiovisual, prensa escrita, etc.), que son nuestra fuente de contenido principal y la base desde la que parte todo.
+
+Aún así, trataremos que desde que nuestra audiencia entre a nuestra página web, hasta el momento que encuentre lo que ha venido a buscar, el número de clicks sea el mínimo.
+
+
+
+## DIAGRAMACIÓN
+A la hora de realizar los diagramas hemos creído conveniente crear cuatro dependiendo del usuario y de la forma en la que accederán a la web.
+
+Primero hemos creado un diagrama general donde el usuario deberá especificar que tipo de usuario es, ya que existen tres formas de acceder y por lo tanto tres tipos de usuarios:
+
+![Diagrama de flujo general](/3-estructura/General.png) 
+
+**1.	Acceder:** son aquellos usuarios que no tienen cuenta y tienen un acceso restringido ya que no podrán de todas las opciones que ofrece la web.
+
+![Diagrama de flujo acceder](/3-estructura/Acceder.png)
+
+**2.	Log in:** son aquellos usuario que ya se han registrado previamente y por lo tanto tienen acceso a todos los servicios que ofrecemos. 
+
+![Diagrama de flujo log in](/3-estructura/Login.png)
+
+**3.	Registrarse:** son aquellos usuarios que deciden unirse a nuestra web por primera vez y una vez registrados podrán obtener su log in y tener un acceso ilimitado al contenido de la web. 
+
+![Diagrama de flujo registrate](/3-estructura/Registrate.png)
+
```

Comment:
No aplicáis además el "Vocabulario visual para arquitectura de información y diseño de la interacción" que corresponde en este apartado. Aquí tenéis una aproximación muy rápida y superficial de cómo podría ser vuestro diagrama: 
![img_20181205_124413819](https://user-images.githubusercontent.com/738823/49511932-ce985600-f88c-11e8-9bcf-1225f87ecfe0.jpg)